docker/README.md

```diff
@@ -1,14 +1,15 @@
 ### Building Redox using Docker images with the toolchain
 
-*All you need is git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
+*All you need is rustup, git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
```

Comment:
Oh, I see, I totally missed it. Maybe we should replace `make pull` with 
```bash
git pull --rebase --recurse-submodules
git submodule sync
git submodule update --recursive --init 
``` 
in the README, since all we need here is to pull all the submodules? There's no actual need for rust to be installed on the host system.